### DISCUSSION TITLE
Request--> Exalted versus World of Darkness

### DISCUSSION THREAD

**Discussion by AMARDA-11** [JohanFalt/Foundry_WoD20#544] [Q&A] [answered]
Is it possible to get a character sheet and stuff for that fan made pdf?

**Comment by JohanFalt** [comment]
No that shouldn't be something I can pull off. You are better of actual looking for other sulutions. I know there are modules that wrap pdf-files in Foundry and I think those are your best bet.

**Reply by AMARDA-11** [reply]
I was merely asking for a character sheet that added a thing like Essence and a Mote counter like Blood Points.

I thought something like that would be pretty easy? Or is it harder then that?

**Reply by JohanFalt** [reply]
If it is only to have a counter then you can use the creature sheet. With that you can under the cogwheel add a bunch of advantages to your liking.

mortal sheet has a bunch as well but not essence.

**Reply by AMARDA-11** [reply]
What about the ability to add custom attributes or the like?

Like, all I need from the sheet is the ability to have two additional tracks, a 1 to 5 dot track of Essence, then a number of boxes that I can label as Motes.

I have mostly figured it out on my end, but it's janky thing. Just, something that might be easier?

https://mrgone.rocksolidshells.com/pdf/Exalted/ExaltedvsWoD_2-Page_Interactive.pdf

Like, in that link, under Willpower, that's really all I need, the Essence and Essence Pool trackers. Something like that a possible feature for future updates? Or to hard to implement?

**Comment by JohanFalt** [comment]
No that I won’t do as the main WoD doesn’t support that.

wouldn’t it be easier to use the exalted system?
https://foundryvtt.com/packages/exaltedthird

**Comment by AMARDA-11** [comment]
I am using this:

https://holdenshearer.wordpress.com/2021/10/26/exalted-vs-world-of-darkness-revised/

It's effectively more a splat that adds Exalted into WoD, so I want the extra options for running that fansplat in the system. Going with the Exalted one is harder due to the fact that I am using WoD for 90% of the rest of the material.

**Reply by JohanFalt** [reply, marked as answer]
Will add Exalted vs WoD revised in next version.
